profile.example.ps1

```diff
@@ -1,25 +1,32 @@
-Push-Location (Split-Path -Path $MyInvocation.MyCommand.Definition -Parent)
-
-# Load posh-git module from current directory
-Import-Module .\posh-git
-
-# If module is installed in a default location ($env:PSModulePath),
-# use this instead (see about_Modules for more information):
-# Import-Module posh-git
-
+# Import the posh-git module, first via installed posh-git module.
+# If the module isn't installed, then attempt to load it from the cloned posh-git Git repo.
+if (Get-Module posh-git -ListAvailable) {
+    Import-Module posh-git
+}
+elseif (Test-Path -LiteralPath $PSScriptRoot\posh-git.psd1) {

```

Comment:
Commenting to help me remember... If this file doesn't go away with 1.0 we should use `Join-Path` here.